test/mocha/35-operations-api.js

```diff
@@ -76,29 +72,25 @@ describe('Operations API', () => {
           await Ed25519VerificationKey2020.from(mockData.keys.authorized);
         const operation = await helpers.signDocument({
           doc: testOperation,
-          verificationMethod:
-            'did:v1:uuid:53ebca61-5687-4558-b90a-03167e4c2838#keys-144',
           key
         });
         await ledgerNode.operations.add({operation});
       });
       it('should fail add operation with an incorrect context', async () => {
         const testOperation = {
-          '@context': 'https://w3id.org/test/v1',
+          '@context': constants.TEST_CONTEXT_V1_URL,
           type: 'CreateWebLedgerRecord',
           creator: 'https://example.com/someCreatorId',
           record: {
-            '@context': 'https://schema.org/',
+            '@context': constants.TEST_CONTEXT_V1_URL,
             id: 'urn:uuid:' + uuid(),
-            value: uuid()
+            name: uuid()
```

Comment:
@aljones15 Can you expand on the property name change?

Reply:
We're using this as `name` is a term in the `context` while `value` is not in the `context`.

Reply:
Great, sounds good.